lib/lib-table-cluster/src/main/java/org/sagebionetworks/table/cluster/TableIndexDAOImpl.java

```diff
@@ -949,4 +959,18 @@ public long getSumOfFileSizes(List<Long> rowIds) {
 		return sum;
 	}
 
+	@Override
+	public List<SynapseStorageProjectStats> getSynapseStorageStats() {

```

Comment:
void streamSynapseStorageStats(Callback<SynapseStorageProjectStats> callback) {
   //callback method should write directly to the CSV
}